screens/BibleScreen.js

```diff
@@ -22,26 +24,42 @@ class BibleScreen extends React.Component {
     },
   };
 
+  componentWillMount() {
+    if (!this.props.passage) {
+      this.props.loadPassage();
+    }
+  }
+
   render() {
-    return (
-      <View style={styles.container}>
-        <ScrollView>
-          { this.props.paragraphs.map(paragraph => (
-            <Paragraph key={paragraph.id} paragraph={paragraph}/>
-          )) }
-        </ScrollView>
-      </View>
-    );
+    if (this.props.passage) {
```

Comment:
I learned that **props** are consistant values, **state** are changing values, should we check this.state.IsDataReady?